airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -151,7 +544,10 @@ def _extract_job(line):
         # [EWT-361] : Fixes out of date regex to extract job id
         matched_job = JOB_ID_PATTERN.search(line or '')
         if matched_job:
-            return matched_job.group('job_id_java') or matched_job.group('job_id_python')
+            job_id = matched_job.group('job_id_java') or matched_job.group('job_id_python') or matched_job.group('job_id_url')
+            logging.info(f"Job id found from the line: [{line}]")
```

Comment:
let use self.log instead of logging

Reply:
its a static fn..there is no self obj present in the args.